tests/bluetooth/bsim_bt/bsim_test_audio/src/unicast_server_test.c

```diff
@@ -34,6 +34,15 @@ static const struct bt_data unicast_server_ad[] = {
 CREATE_FLAG(flag_connected);
 CREATE_FLAG(flag_stream_configured);
 
+static void print_ase_info(struct bt_audio_ep *ep, void *user_data)
+{
+	struct bt_audio_ep_info info;
+
+	bt_audio_ep_get_info(ep, &info);
+	printk("ASE info: id %u state %u dir %u\n", info.id, info.state,
+	       info.dir);
+}
```

Comment:
Suggest to also add this to the shell `subsys/bluetooth/shell/audio.c` with a command to print ASE info